Kernel/Devices/PATAChannel.cpp

```diff
@@ -419,36 +401,31 @@ bool PATAChannel::ata_read_sectors(u32 start_sector, u16 count, u8* outbuf, bool
     ASSERT(count <= 256);
     LOCKER(s_lock());
 #ifdef PATA_DEBUG
-    kprintf("%s(%u): PATAChannel::ata_read_sectors request (%u sector(s) @ %u into %p)\n",
-        Process::current->name().characters(),
-        Process::current->pid(),
-        count,
-        start_sector,
-        outbuf);
+    dbg() << Process::current->name().characters() << "(" << Process::current->pid() << "): PATAChannel::ata_read_sectors request (" << count << " sector(s) @ " << start_sector << " into " << outbuf << ")";
```

Comment:
I replaced kprintf with dbg call, then I changed the duplicated data in a different commit.
I think it's OK since one can see the logical "steps" towards a complete results in the commits.